spacy/tests/test_misc.py

```diff
@@ -81,32 +78,33 @@ def test_util_get_package_path(package):
     assert isinstance(path, Path)
 
 
-def test_PrecomputableAffine(nO=4, nI=5, nF=3, nP=2):
-    model = PrecomputableAffine(nO=nO, nI=nI, nF=nF, nP=nP).initialize()
-    assert model.get_param("W").shape == (nF, nO, nP, nI)
-    tensor = model.ops.alloc((10, nI))
-    Y, get_dX = model.begin_update(tensor)
-    assert Y.shape == (tensor.shape[0] + 1, nF, nO, nP)
-    dY = model.ops.alloc((15, nO, nP))
-    ids = model.ops.alloc((15, nF))
-    ids[1, 2] = -1
-    dY[1] = 1
-    assert not model.has_grad("pad")
-    d_pad = _backprop_precomputable_affine_padding(model, dY, ids)
-    assert d_pad[0, 2, 0, 0] == 1.0
-    ids.fill(0.0)
-    dY.fill(0.0)
-    dY[0] = 0
-    ids[1, 2] = 0
-    ids[1, 1] = -1
-    ids[1, 0] = -1
-    dY[1] = 1
-    ids[2, 0] = -1
-    dY[2] = 5
-    d_pad = _backprop_precomputable_affine_padding(model, dY, ids)
-    assert d_pad[0, 0, 0, 0] == 6
-    assert d_pad[0, 1, 0, 0] == 1
-    assert d_pad[0, 2, 0, 0] == 0
+# @pytest.mark.skip(reason="No precomputable affine")
+# def test_PrecomputableAffine(nO=4, nI=5, nF=3, nP=2):
+#     model = PrecomputableAffine(nO=nO, nI=nI, nF=nF, nP=nP).initialize()
+#     assert model.get_param("W").shape == (nF, nO, nP, nI)
+#     tensor = model.ops.alloc((10, nI))
+#     Y, get_dX = model.begin_update(tensor)
+#     assert Y.shape == (tensor.shape[0] + 1, nF, nO, nP)
+#     dY = model.ops.alloc((15, nO, nP))
+#     ids = model.ops.alloc((15, nF))
+#     ids[1, 2] = -1
+#     dY[1] = 1
+#     assert not model.has_grad("pad")
+#     d_pad = _backprop_precomputable_affine_padding(model, dY, ids)
+#     assert d_pad[0, 2, 0, 0] == 1.0
+#     ids.fill(0.0)
+#     dY.fill(0.0)
+#     dY[0] = 0
+#     ids[1, 2] = 0
+#     ids[1, 1] = -1
+#     ids[1, 0] = -1
+#     dY[1] = 1
+#     ids[2, 0] = -1
+#     dY[2] = 5
+#     d_pad = _backprop_precomputable_affine_padding(model, dY, ids)
+#     assert d_pad[0, 0, 0, 0] == 6
+#     assert d_pad[0, 1, 0, 0] == 1
+#     assert d_pad[0, 2, 0, 0] == 0
```

Comment:
Can this be uncommented?

Reply:
`PrecomputableAffine` doesn't exist as a separate layer anymore, so I removed this.